static/app/views/dashboards/widgetBuilder/components/visualize.tsx

```diff
@@ -162,39 +180,80 @@ function Visualize() {
                     />
                     {/* TODO: Handle aggregates with multiple parameters */}
                     <AggregateCompactSelect
-                      options={aggregateOptions.map(option => ({
-                        value: option.value.meta.name,
-                        label: option.value.meta.name,
-                      }))}
+                      options={
+                        isChartWidget
+                          ? aggregateOptions
+                          : [NONE_AGGREGATE, ...aggregateOptions]
+                      }
                       value={parseFunction(stringFields?.[index] ?? '')?.name ?? ''}
                       onChange={aggregateSelection => {
```

Comment:
This is _starting_ to get a little hairy, but I want to see what it looks like when we get all of the fields supported and see what we can refactor later on. If all of the hairiness can be contained to here, I'm not too concerned with it being more complex